AsyncDisplayKit/TextKit/ASTextKitFontSizeAdjuster.mm

```diff
@@ -104,8 +106,14 @@ - (NSUInteger)lineCountForString:(NSAttributedString *)attributedString
 
 - (CGFloat)scaleFactor
 {
+  if (_measured) {
+    return _scaleFactor;
+  }
+  
   if ([_attributes.pointSizeScaleFactors count] == 0 || isinf(_constrainedSize.width)) {
-    return 1.0;
+    _measured = YES;
```

Comment:
Where is _measured set to NO?  Do we destroy and recreate this object if the attributed string or constrainedSize changes?